BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-117: Create login screen

- Create Auth actions for Login
- Create Auth reducer for Login
- Create Auth constants for Login
- Update Employee redux and bring token from server
- Modify employee component
- Delete header and footer in employee home page
- Display modal confirmation for logout

QA:
How to test?
To test this ticket we should open the link of the app deployed in Vercel. Then, we should verify the correct functionality of:

Logging in correctly with an employee user
-Email: [flune19@desdev.cn](mailto:flune19@desdev.cn)
-Password: ioAgbm27rl
(You can check the database to try other employees as well: [Database: employees](https://cloud.mongodb.com/v2/6346f3df659ee31f0aaca3d1#metrics/replicaSet/63470669f3e4223d9265680f/explorer/test/employees/find))

Have access to sign up form.

After that, you should see a list of the projects in which the logged employee is assigned (if the employee is not working on any projects you should see a message saying that)

The token of the employee is shown at the console, so you can still do some tests with that employee in Postman.

You should be able to add hours, create a new timesheet and update the employee.

In the sidebar, you are able to Log-out and a modal confirming the logout must be shown
